new-lamassu-admin/src/pages/Blacklist/Blacklist.styles.js

```diff
@@ -0,0 +1,42 @@
+export default {
+  grid: {
+    flex: 1,
+    height: '100%'
+  },
+  content: {
+    display: 'flex',
+    flexDirection: 'column',
+    flex: 1,
+    marginLeft: 48
+  },
+  bottomRight: {
+    marginLeft: 'auto',
+    marginTop: 'auto',
+    marginBottom: '24px'
+  },
+  modalTitle: {
+    lineHeight: '120%',
+    color: '#1b2559',
+    fontSize: 14,
+    fontFamily: 'Mont',
+    fontWeight: 900
+  },
```

Comment:
Isn't this already defined in typography? We don't want to manually set font styles nor use unnamed colors. The colors are all exported with names from the styling/variables file.